ci-operator/step-registry/openshift/microshift/e2e/test-rpm-upgrade/openshift-microshift-e2e-test-rpm-upgrade-commands.sh

```diff
@@ -0,0 +1,118 @@
+#!/bin/bash
```

Comment:
I'll TAL and let you know

Reply:
@dhellmann Moving the script to the main repo is totally doable.  It's going to create some inconsistency in our testing pipeline, since the non-metal tests are all defined in openshift/release.  Moving these ec2 tests to the microshift repo is a good idea, but we should hold off on that until we have a good strategy for doing so.